Simplenote.xcodeproj/project.pbxproj

```diff
@@ -133,6 +133,7 @@
 		B50789FE1C1F5517009F097A /* SPInteractivePushPopAnimationController.m in Sources */ = {isa = PBXBuildFile; fileRef = 17C421CE1BE0BAB100752B56 /* SPInteractivePushPopAnimationController.m */; };
 		B50789FF1C1F5592009F097A /* markdown-default.css in Resources */ = {isa = PBXBuildFile; fileRef = 1748382E1BC1CC2D00E834AF /* markdown-default.css */; };
 		B5078A001C1F5595009F097A /* markdown-dark.css in Resources */ = {isa = PBXBuildFile; fileRef = 17BC3D461BC46BA800535DF3 /* markdown-dark.css */; };
+		B50E99F4234242130092F274 /* UISearchBar+Simplenote.swift in Sources */ = {isa = PBXBuildFile; fileRef = B50E99F3234242130092F274 /* UISearchBar+Simplenote.swift */; };
```

Comment:
these are autogenerated? I feel like I see them in every PR.

Reply:
Yup, handled by Xcode!

Reply:
I'm not sure if we can do this easily or not, but in our JavaScript PRs we were having ugly diff noise from things like the `package-lock.json` file which is very similar to this - auto-generated and un-reviewable practically speaking.

I wonder if we can set some switch to hide these diffs from the review screen? As in, start collapsed with the `See changes` link to expand it…

Reply:
lol: https://github.com/marcelofabri/pbxproj-diff-hider

Reply:
Personally I would never try to hide these diffs @dmsnell !!! Anything that breaks in there (bad merge, whatever) might render the entire project unusable.

We do actually check on those changes, and make sure files are not added twice (for instance). And it's especially useful when merging branches!